packages/stylable.io/docs/references/imports.md

```diff
@@ -6,39 +6,95 @@ layout: docs
+
+```css
+:import {
+    -st-from: './stylesheet.st.css';
+    -st-default: <DEFAULT_NAME>;
+    -st-named: <NAMED_PART1, NAMED_PART2, ...>;
+}
+```
+
+Every example below will feature both types of import syntaxes, their end result is identical.
 
 > **Note**  
 > * `:import` is a Stylable directive and not a selector.
-> * Using `import` as part of a complex selector or inside a CSS ruleset does not import.
-> * Multiple imports may conflict and the last one in the file wins.
+> * likewise, `@st-import` is a Stylable directive and not an actual at-rule.
+> * Import statements cannot be used as a part of a complex selector or inside a CSS ruleset.
+> * Multiple imports may conflict in their used symbol names; the last one in the file wins.
 
 ## Imports - basic usage
 
 Here are some examples of how you can use imports in your **Stylable** stylesheet.
 
 ### Import the default export of a local reference stylesheet for use in the scoped stylesheet
 
-Import the `toggle-button.css` stylesheet from a local location. Assign the name `ToggleButton` to the default export of that stylesheet for use in this scoped stylesheet. 
+Import the `button.st.css` stylesheet from a local location. Assign the name `Button` to the default export of that stylesheet for use in this scoped stylesheet. 
+
+When importing another stylesheet, the default import represents the root of the stylesheet and is generally treated as a component, and named imports represent other internal stylesheet parts.
 
 > **Note**  
-> Generally when importing a **default** value from a css file, you can use a capital letter to signify that the value is used as a component in this stylesheet. 
+> Generally when importing a **default** value from a stylable file, you should use a capital letter to signify that the value represents a component root node in this stylesheet. 
+
 
 ```css
-/* CSS */
+/* comp.st.css - new syntax */
+@st-import ToggleButton from './button.st.css';
+```
+
+```css
+/* comp.st.css - old syntax */
 :import {
-    -st-from: './toggle-button.css';
+    -st-from: './button.st.css';
     -st-default: ToggleButton;
 }
 ```
 
 ```js
-/* ES6 equivalent */
-import ToggleButton from './toggle-button.css';
+/* ES6 "equivalent" */
+import ToggleButton from './button.st.css';
+```
+
+### Import named parts from a local stylesheet
+Named imports from a stylesheet can be used to bring symbols of different types, which you can then use inside your stylesheet.
+
+- Elements
+- Classes
+- Stylable Variables
+- CSS Variables
+- Keyframes - special case, see [importing keyframes](#importing_keyframes) entry below
+
+In this 
+```css
+/* comp.st.css - new syntax */
+@st-import [label, icon, --bgColor] from './button.st.css';
+```
+
+```css
+/* comp.st.css - old syntax */
+:import {
+    -st-from: './button.st.css';
+    -st-named: label, icon, --bgColor;
+}
+```
+
+```js
+/* ES6 "equivalent" */
+import { label, icon, --bgColor } from './button.st.css';
```

Comment:
I think that the es6 equivalents examples can be removed because they might suggest that importing an `st.css` file might resolve in that way - which it doesn't.